code/_onclick/click.dm

```diff
@@ -515,8 +515,8 @@ if(selected_ability.target_flags & flagname){\
 
 
 // Simple helper to face what you clicked on, in case it should be needed in more than one place
-/mob/proc/face_atom(atom/A)
-	if(buckled || stat != CONSCIOUS || !A || !x || !y || !A.x || !A.y)
+/atom/proc/face_atom(atom/A)
+	if(!A || !x || !y || !A.x || !A.y)
 		return
```

Comment:
why is this not on movable?

Reply:
because the other proc /mob/face_atom(atom/A) was already there, i just tweaked it a bit.

If the question is why it's not /atom/moveable/proc/face_atom, well it's because the xeno turrets is not a moveable AFAIK